src/clr-angular/typography/_properties.typography.scss

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
+// This software is released under MIT license.
+// The full license information can be found in LICENSE in the root directory of this project.
+
+@include exports('typography.properties') {
+  @if $clr-use-custom-properties == true {
+    :root {
+      --clr-font: var(#{$clr-font});
```

Comment:
should this be ` --clr-font: var(--clr-font);`?

Reply:
No. We still key off the SASS variables at the topmost level of the custom properties in many cases. Defining it as `--clr-font: var(--clr-font);` is just self-referential. It would be given no value.

Reply:
D'oh.. how did I miss the self-referential thing. 🤦‍♀ Thanks! 👍